app/Http/Requests/Course/UpdateLesson.php

```diff
@@ -24,8 +24,9 @@ public function authorize()
 	public function rules()
 	{
 		return [
-			'name'   => 'string',
-			'groups' => 'numeric',
+			'name' => 'required|string',
+			'group_id' => 'required|numeric',
```

Comment:
maybe something like this makes sense here: https://laravel.com/docs/5.7/validation#rule-exists

Reply:
I think it would be:
```suggestion
			'group_id' => 'required|numeric|exists:groups,id',
```